.github/workflows/build.yaml

```diff
@@ -0,0 +1,32 @@
+name: Build Eleventy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+  
+    strategy:
+      matrix:
+        node-version: [16.x]
```

Comment:
once you upgrade to 22.04 would suggest moving up to 18.x or 20.x https://endoflife.date/nodejs